### ISSUE TITLE
No such file or directory /pkgs/mkl-2017.0.3-0/info/has_prefix'

### ISSUE THREAD

**Issue by symdeb** [conda/conda#9375]
Ubuntu 19.10 

conda create -n tensorflow_env tensorflow python=3.5
Fetching package metadata ...........
Solving package specifications: .

Package plan for installation in environment /media/user/H3D1P2/Tools/anaconda3/envs/tensorflow_env:

The following NEW packages will be INSTALLED:

    absl-py:           0.5.0-py_0           conda-forge
    backports:         1.0-py_2             conda-forge
    backports.weakref: 1.0.post1-py35_0     conda-forge
    blas:              1.0-mkl                         
    bleach:            1.5.0-py35_0         conda-forge
    ca-certificates:   2019.9.11-hecc5488_0 conda-forge
    certifi:           2018.8.24-py35_1001  conda-forge
    html5lib:          0.9999999-py35_0     conda-forge
    libprotobuf:       3.5.2-0              conda-forge
    markdown:          3.1.1-py_0           conda-forge
    mkl:               2017.0.3-0                      
    ncurses:           5.9-10               conda-forge
    numpy:             1.13.1-py35_0                   
    openssl:           1.0.2o-0             conda-forge
    pip:               18.0-py35_1001       conda-forge
    protobuf:          3.5.2-py35_0         conda-forge
    python:            3.5.5-1              conda-forge
    readline:          7.0-0                conda-forge
    setuptools:        40.4.3-py35_0        conda-forge
    six:               1.11.0-py35_1        conda-forge
    sqlite:            3.20.1-2             conda-forge
    tensorboard:       1.5.1-py35_1         conda-forge
    tensorflow:        1.5.1-py35_0         conda-forge
    tk:                8.6.8-0              conda-forge
    webencodings:      0.5.1-py_1           conda-forge
    werkzeug:          0.16.0-py_0          conda-forge
    wheel:             0.32.0-py35_1000     conda-forge
    xz:                5.2.3-0              conda-forge
    zlib:              1.2.11-0             conda-forge

Proceed ([y]/n)? y

python-3.5.5-1 100% |##############################################################| Time: 0:00:03   6.06 MB/s
certifi-2018.8 100% |##############################################################| Time: 0:00:00   7.38 MB/s
six-1.11.0-py3 100% |##############################################################| Time: 0:00:00  16.32 MB/s
absl-py-0.5.0- 100% |##############################################################| Time: 0:00:00   7.40 MB/s
backports.weak 100% |##############################################################| Time: 0:00:00  11.05 MB/s
html5lib-0.999 100% |##############################################################| Time: 0:00:00   7.08 MB/s
setuptools-40. 100% |##############################################################| Time: 0:00:00   7.56 MB/s
bleach-1.5.0-p 100% |##############################################################| Time: 0:00:00   7.65 MB/s
protobuf-3.5.2 100% |##############################################################| Time: 0:00:00   4.96 MB/s
wheel-0.32.0-p 100% |##############################################################| Time: 0:00:00  15.85 MB/s
pip-18.0-py35_ 100% |##############################################################| Time: 0:00:01   1.31 MB/s
tensorboard-1. 100% |##############################################################| Time: 0:00:00   5.37 MB/s
tensorflow-1.5 100% |##############################################################| Time: 0:00:05   6.64 MB/s
An unexpected error has occurred.
Please consider posting the following information to the
conda GitHub issue tracker at:

    https://github.com/conda/conda/issues

Current conda install:

               platform : linux-64
          conda version : 4.3.16
       conda is private : False
      conda-env version : 4.3.16
    conda-build version : 3.18.9
         python version : 3.7.4.final.0
       requests version : 2.22.0
       root environment : /media/user/H3D1P2/Tools/anaconda3  (writable)
    default environment : /media/user/H3D1P2/Tools/anaconda3
       envs directories : /media/user/H3D1P2/Tools/anaconda3/envs
                          /home/user/.conda/envs
          package cache : /media/user/H3D1P2/Tools/anaconda3/pkgs
                          /home/user/.conda/pkgs
           channel URLs : https://conda.anaconda.org/conda-forge/linux-64
                          https://conda.anaconda.org/conda-forge/noarch
                          https://repo.continuum.io/pkgs/free/linux-64
                          https://repo.continuum.io/pkgs/free/noarch
                          https://repo.continuum.io/pkgs/r/linux-64
                          https://repo.continuum.io/pkgs/r/noarch
                          https://repo.continuum.io/pkgs/pro/linux-64
                          https://repo.continuum.io/pkgs/pro/noarch
            config file : /home/user/.condarc
           offline mode : False
             user-agent : conda/4.3.16 requests/2.22.0 CPython/3.7.4 Linux/5.3.0-19-lowlatency debian/buster/sid glibc/2.30
                UID:GID : 1000:1000

`$ /media/user/H3D1P2/Tools/anaconda3/bin/conda create -n tensorflow_env tensorflow python=3.5`

    Traceback (most recent call last):
      File "/home/user/.local/lib/python3.7/site-packages/conda/gateways/disk/read.py", line 44, in yield_lines
        with open(path) as fh:
    FileNotFoundError: [Errno 2] No such file or directory: '/media/user/H3D1P2/Tools/anaconda3/pkgs/mkl-2017.0.3-0/info/has_prefix'
    
    During handling of the above exception, another exception occurred:
    
    Traceback (most recent call last):
      File "/home/user/.local/lib/python3.7/site-packages/conda/gateways/disk/read.py", line 52, in yield_lines
        raise StopIteration
    StopIteration
    
    The above exception was the direct cause of the following exception:
    
    Traceback (most recent call last):
      File "/home/user/.local/lib/python3.7/site-packages/conda/cli/install.py", line 357, in install
        execute_actions(actions, index, verbose=not context.quiet)
      File "/home/user/.local/lib/python3.7/site-packages/conda/plan.py", line 809, in execute_actions
        execute_instructions(plan, index, verbose)
      File "/home/user/.local/lib/python3.7/site-packages/conda/instructions.py", line 247, in execute_instructions
        cmd(state, arg)
      File "/home/user/.local/lib/python3.7/site-packages/conda/instructions.py", line 107, in UNLINKLINKTRANSACTION_CMD
        txn = UnlinkLinkTransaction.create_from_dists(index, prefix, unlink_dists, link_dists)
      File "/home/user/.local/lib/python3.7/site-packages/conda/core/link.py", line 124, in create_from_dists
        for dist, pkg_dir in zip(link_dists, pkg_dirs_to_link))
      File "/home/user/.local/lib/python3.7/site-packages/conda/core/link.py", line 124, in <genexpr>
        for dist, pkg_dir in zip(link_dists, pkg_dirs_to_link))
      File "/home/user/.local/lib/python3.7/site-packages/conda/gateways/disk/read.py", line 84, in read_package_info
        paths_data = read_paths_json(extracted_package_directory)
      File "/home/user/.local/lib/python3.7/site-packages/conda/gateways/disk/read.py", line 150, in read_paths_json
        has_prefix_files = read_has_prefix(join(info_dir, 'has_prefix'))
      File "/home/user/.local/lib/python3.7/site-packages/conda/gateways/disk/read.py", line 200, in read_has_prefix
        return {pr.filepath: (pr.placeholder, pr.filemode) for pr in parsed_lines}
      File "/home/user/.local/lib/python3.7/site-packages/conda/gateways/disk/read.py", line 200, in <dictcomp>
        return {pr.filepath: (pr.placeholder, pr.filemode) for pr in parsed_lines}
      File "/home/user/.local/lib/python3.7/site-packages/conda/gateways/disk/read.py", line 199, in <genexpr>
        parsed_lines = (parse_line(line) for line in yield_lines(path))
    RuntimeError: generator raised StopIteration
    
    During handling of the above exception, another exception occurred:
    
    Traceback (most recent call last):
      File "/home/user/.local/lib/python3.7/site-packages/conda/exceptions.py", line 626, in conda_exception_handler
        return_value = func(*args, **kwargs)
      File "/home/user/.local/lib/python3.7/site-packages/conda/cli/main.py", line 134, in _main
        exit_code = args.func(args, p)
      File "/home/user/.local/lib/python3.7/site-packages/conda/cli/main_create.py", line 68, in execute
        install(args, parser, 'create')
      File "/home/user/.local/lib/python3.7/site-packages/conda/cli/install.py", line 372, in install
        raise CondaRuntimeError('RuntimeError: %s' % e)
    conda.exceptions.CondaRuntimeError: Runtime error: RuntimeError: generator raised StopIteratio

**Comment by msarahan** [comment]
That is an old, unsupported conda version. If you update and still have issues, we can take a look. Cons 4.7.12 is current. Also, we dropped python 3.5 when 3.7 came out. The latest tensorflow packages are only available for 3.6 and 3.7.

**Comment by symdeb** [comment]
I got this with the 19.10 Linux nstall of anaconda and could not fix it in any way. It looks like Python 3.7 was corrupt or so. Re-installed Ubuntu 19.10 from scratch and problem is gone. The only thing was an numpy error that is was too old and 
conda uninstall numpy
conda install numpy
fix it.
I was advised to use 19.04 and conda create -n tensorflow-3.5 tensorflow python-3.5
For a priject we have using tensor-flow
I used anaconda 19.10 and python 3.5 and "import tensorflow as tf" worked w/o errors
But should this be python-3.7 now ?
Thanks
